src/install/install.zig

```diff
@@ -10545,6 +10547,23 @@ pub const PackageManager = struct {
                             builder,
                         );
 
+                        // Update workspace paths
+                        manager.lockfile.workspace_paths.deinit(manager.lockfile.allocator);
+                        manager.lockfile.workspace_paths = try lockfile.workspace_paths.clone(manager.lockfile.allocator);
```

Comment:
Small optimization: we can `ensureTotalCapacity` instead of `clone` and update both keys and values in the for loop, avoiding an extra memcpy